front_end/src/components/layout/Navbar.js

```diff
@@ -2,10 +2,12 @@ import React, { Component } from "react";
 import { Link } from "react-router-dom";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
-import { logoutUser } from "../../actions/authActions";
+import { logoutUser,getCurrentUser } from "../../actions/authActions";
 import { clearCurrentProfile } from "../../actions/profileActions";
 import AnchorLink from "react-anchor-link-smooth-scroll";
 import NavbarMenu from "./NavbarMenu";
+import axios from "axios";
+import jwt_decode from "jwt-decode";
```

Comment:
is jwt_decode an alternative to redux store?

Reply:
iirc, it's used to distributed authentication tokens to logged in users.

Reply:
but in this case, does it include only the token or all the other info (name, email, points)?